### ISSUE TITLE
SCP/SFTP support does not work correctly

### ISSUE THREAD

**Issue by jpstotz** [TeamAmaze/AmazeFileManager#2843]
**Describe the bug**
I have set-up an SCP connection to a Linux system, Amaze is able to connect to it and displays the initial directory content, but it is not able to display the content of sub-folders, every selected folder remains empty. 

Additionally if you select a file or folder and execute the "Copy" function nothing happens (no "persistent toast" is shown that allows you to copy a file to a different directory.

Also viewing a file on a SCP mount is not possible, if you do that Chrome opens on the address `127.0.0.1:7871/home 

From logcat I can see that `sftp subsystem`  is used.

**To Reproduce**
1. Set-up an SCP connection to a Linux system using username and key file
2. Select the SCP entry, enter the key password
3. The file and folder list from the Linux server is shown
4. Select a folder that contains files
5. The folder is opened but no files are displayed inside

Logcat does not show any exceptions (just the common failed attempts to load the ssh keyfile for the wrong format).

**Smartphone (please complete the following information):**
 - Device: Tablet FullHD
 - OS: Android 9
 - Rooted: No
 - Amaze Version: 3.6.1 from Google Play Store

**Comment by TranceLove** [comment]
May I know the SSH server implementation? Is it just a typical x86 box or a Raspberry Pi (or similar)?

**Comment by jpstotz** [comment]
Nothing special, just an OpenSSH 1.7.9 server on Debian buster (Raspberry).

**Comment by jpstotz** [comment]
I made some more tests and noticed that this bug always happens if I specify an default path. If I don't specify an optional path it seems to work.

This is just a guess but this may be related to the fact that this optional path is included into the database sftp entry column `path`. I checked the utilities.db and found in table sftp column `path` the entry  `ssh://user@host:22/optional/path` which seems to prevent Amaze from retrieving it for subfolder and files. At least I can see a lot of log entries like 
`Error getting auth private key for URI [ssh://...]  androidx.room.EmptyResultSetException: Query returned empty result set: SELECT ssh_key FROM sftp WHERE PATH = ?`

**Comment by TranceLove** [comment]
Verified, and problem occurs when password authentication is used.

**Comment by YlanAllouche** [comment]
Using a regular x86 SSH server, it works only the first time for me, with or without default path.
Once the app is closed, I need to re-enter everything or modify the existing entry.

**Comment by jpstotz** [comment]
@YlanAllouche After performing some more tests I got to the same conclusion. If you add an SCP entry everything works as expected but as soon as you restart the app the SCP entry no longer fully works (only shows the home directory content not no other directory content).

**Comment by VishalNehra** [comment]
> @YlanAllouche After performing some more tests I got to the same conclusion. If you add an SCP entry everything works as expected but as soon as you restart the app the SCP entry no longer fully works (only shows the home directory content not no other directory content).

Yes. We have already pinpointed the cause, you can expect a fix with upcoming hotfix in few days. cc @TranceLove

**Comment by TranceLove** [comment]
@VishalNehra Yes, I'm on it. Stay tuned😉
